docs/user-guide/configuration/workflow.md

```diff
@@ -115,6 +117,31 @@ If job names are prefixed with tildes in a `requires` line, then the job will st
 
 is equivalent to the Boolean expression `A OR C OR E OR (B AND D AND F)`. Such a complicated `requires` line in an actual workflow should be regarded as a code smell.
 
+## Branch filtering
+To trigger jobs in your pipeline after a specific branch is committed, you can use branch filtering jobs. The format is `~commit:branchName`. Also you can use regex filter after `~commit:` (e.g. `~commit:/^feature-.*$/`).  
```

Comment:
Do you also need to add `~pr:`here like in your design PR (https://github.com/screwdriver-cd/screwdriver/pull/1166)?

Reply:
@tkyi added!